launch/docker.go

```diff
@@ -114,3 +124,88 @@ func (d *docker) execDockerCommand(args ...string) error {
 	}
 	return nil
 }
+
+func (d *docker) kill(sig os.Signal) {
+	killedCmds := make([]*exec.Cmd, 0, 10)
+
+	for _, v := range d.commands {
+		var err error
+		d.mutex.Lock()
+		if v.ProcessState != nil {
+			continue
+		}
+		d.mutex.Unlock()
+
+		if d.useSudo {
+			cmd := execCommand("sudo", "kill", fmt.Sprintf("-%v", signum(sig)), strconv.Itoa(v.Process.Pid))
+			err = cmd.Run()
+		} else {
+			err = v.Process.Signal(sig)
+		}
+
+		if err != nil {
+			logrus.Warn(fmt.Errorf("failed to stop process: %v", err))
+		} else {
+			killedCmds = append(killedCmds, v)
+		}
+	}
+
+	err := d.waitForProcess(killedCmds)
+	if err != nil {
+		logrus.Warn(err)
+	}
+}
+
+func (d *docker) clean() {
+	err := d.execDockerCommand("volume", "rm", "--force", d.volume)
+
+	if err != nil {
+		logrus.Warn(fmt.Errorf("failed to remove volume: %v", err))
+	}
+}
+
+func (d *docker) waitForProcess(cmds []*exec.Cmd) error {
+	// Reducing this value will make the test faster.
+	// However, be sure to specify a time when you can sufficiently confirm that the process is dead.
```

Comment:
Do we need this comments?
Now it looks we do not have retry test.

Reply:
"waitaForProcess" is definitely waiting more than a second for a test.
And there are currently two or more tests where "waitForProcess" is used.
If we set this value to 500ms, I think we could reduce the test time by more than a second.

Reply:
Thx. I see.